File: Java/Estrutura condicional/Temperatura.java
import java.util.Locale;
import java.util.Scanner;

public class Temperatura {
    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        char resposta;
        double c, f;

        System.out.print("Voce vai digitar a temperatura em qual escala (C/F)?");
        resposta = sc.next().charAt(0);

        if (resposta == 'C') {
            System.out.print("Digite a temperatura em Celsius: ");
            c = sc.nextDouble();
            f = c * 9 / 5 + 32;
            System.out.print("Temperatura equivalente em Fahrenheit: " + String.format("%.2f", f));
        } else {
            System.out.print("Digite a temperatura em Fahrenheit: ");
            f = sc.nextDouble();
            c = (f - 32) / 1.8;
            System.out.print("Temperatura equivalente em Celsius: " + String.format("%.2f", c));
        }

        sc.close();

    }
}

File: Python/Estrutura Condicional/Temperatura.py
c: float; f: float
resposta: str


resposta = str(input("Voce vai digitar a temperatura em qual escala (C/F)? "))

if resposta == 'f':
    f = float(input("Digite a temperatura em Fahrenheit: "))
    c = (f - 32) / 1.8
    print(f"Temperatura equivalente em Celsius: {c:.2f}")
else:
    c = float(input("Digite a temperatura em Celsius: "))
    f =  c * 9 / 5 + 32
    print(f"Temperatura equivalente em Fahrenheit: {f:.2f}")